docs/user/ovirt/install_ipi.md

```diff
@@ -122,3 +123,20 @@ $ export KUBECONFIG=$PWD/install_dir/auth/kubeconfig
 $ oc get nodes
 ```
 
+### ovirt-config.yaml
+
+The ovirt-config.yaml is created under ${HOME}/.ovirt directory by the installer.
+It contains all information how the installer connects to oVirt and can be re-used
+if required to re-trigger a new installation.
+
+Below the description of all config options in ovirt-config.yaml.
+
+| Name           | Value                          | Type     | Example                                                                                                |
+| ---------------|:------------------------------:|:--------:|:------------------------------------------------------------------------------------------------------:|
+| ovirt_url      | URL for Engine API             | string   | https://engine.fqdn.home/ovirt-engine/api                                                              |
+| ovirt_fqdn     | Engine FQDN                    | string   | engine.fqdn.home                                                                                       |
+| ovirt_username | User to connect with Engine    | string   | ocpadmin@internal                                                                                      |
```

Comment:
Despite that we have to document the minimum roles for the RHV user that will run the installer, we haven't change yet the default user, that for now I think should remain the admin@internal